package.json

```diff
@@ -52,6 +52,10 @@
     "vite": "^4.2.1",
     "vite-plugin-environment": "^1.1.3",
     "vite-plugin-pwa": "^0.14.4",
-    "vitest": "^0.29.7"
+    "vitest": "^0.29.7",
+    "workbox-window": "^6.5.4"
+  },
+  "peerDependencies": {
+    "workbox-window": "^6.5.4"
```

Comment:
I don't think it makes sense to list peerDependencies in our package.json; it only makes sense when this package is installed as a dependency of another package.
```suggestion
```